src/FdrWindow/CMakeLists.txt

```
@@ -3,9 +3,10 @@ add_library(FdrWindow STATIC
     ${CMAKE_CURRENT_SOURCE_DIR}/FdrParameters/FdrParameters.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/FdrOpenGLRender/FdrOpenGLRender.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/FdrGLFW/FdrGLFW.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/FdrImGui/FdrImGui.cpp
 )
 
-target_include_directories(FdrWindow PUBLIC ${CMAKE_SOURCE_DIR}/include)
+target_include_directories(FdrWindow PUBLIC ${CMAKE_SOURCE_DIR}/include ${CMAKE_SOURCE_DIR}/third_party/imgui)
```

```suggestion
target_include_directories(FdrWindow PUBLIC ${CMAKE_SOURCE_DIR}/include
```
Это и так делается при добавлении ImGui как пекеджа в симейке (в /cmake/third_party.cmake)